pkg/minikube/bootstrapper/bsutil/featuregates.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// bsutil package will eventually be renamed to kubeadm package after getting rid of older one
+package bsutil
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+	"k8s.io/kubernetes/cmd/kubeadm/app/features"
+)
+
+// supports indicates whether a feature name is supported on the
+// feature gates for kubeadm
+func supports(featureName string) bool {
```

Comment:
It would be pretty simple, but you could write a unit test for this function :)

Reply:
:D

Reply:
done